packages/cfd/src/Stores/Modules/CFD/Helpers/cfd-config.ts

```diff
@@ -99,13 +99,13 @@ export const getMtCompanies = (is_eu: boolean) => {
             all: {
                 mt5_account_type: all_config.account_type,
                 leverage: all_config.leverage,
-                title: localize('Demo Swap-Free'),
+                title: localize('Swap-Free'),
```

Comment:
@shontzu-deriv These titles are used in some other places in the code(`cfd-reset-password-modal`, ...) and changing them will affect other places as well.